brianlong/www.validators.app
# 181242193_add_columns_to_ping_thing

#### What's this PR do?
- Add column `application`, `commitment_level`, and `success` to ping things
- Update the view and the docs

#### How should this be manually tested?
- Go to /ping-thing
- check column 'application' (if you don't have the data use seeds)
- Use cURL or postman and send POST request with new data, check if it's saved, run process_ping_thing_service and check if PingThing was created

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181242193

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
